changes/8984.misc

```diff
@@ -0,0 +1,2 @@
+Add a new config value for Datastore query
+timeout ckan.datastore.default_query_timeout_ms
```

Comment:
We need to tell users what the new timeout configuration options and default values are so they can change them if needed.